linux_os/guide/system/network/network-iptables/iptables_activation/service_iptables_enabled/rule.yml

```diff
@@ -31,12 +31,22 @@ references:
     nist: AC-4,CM-7(b),CA-3(5),SC-7(21),CM-6(a)
     nist-csf: DE.AE-1,ID.AM-3,PR.AC-5,PR.DS-5,PR.IP-1,PR.PT-3,PR.PT-4
 
-platform: system_with_kernel and package[iptables] and service_disabled[firewalld]
+platform: machine and package[iptables] and service_disabled[firewalld]
```

Comment:
Did you intent go back to `machine` vs `system_with_kernel`?

Reply:
That was a miss during rebase thanks for raising the flag